slack_sdk/web/async_client.py

```diff
@@ -3191,6 +3192,75 @@ async def conversations_replies(
         )
         return await self.api_call("conversations.replies", http_verb="GET", params=kwargs)
 
+    async def conversations_requestSharedInvite_approve(
+        self,
+        *,
+        invite_id: str,
+        channel_id: Optional[str] = None,
+        is_external_limited: Optional[str] = None,
+        message: Optional[Dict[str, Any]] = None,
+        **kwargs,
+    ) -> AsyncSlackResponse:
+        """Approve a request to add an external user to a channel. This also sends them a Slack Connect invite.
+        https://api.slack.com/methods/conversations.requestSharedInvite.approve
+        """
+        kwargs.update(
+            {
+                "invite_id": invite_id,
+                "channel_id": channel_id,
+                "is_external_limited": is_external_limited,
+                "message": message,
+            }
+        )
+        if message is not None:
```

Comment:
When you have these lines of code, the above `"message": message,` is unnecessary

Reply:
Good catch 💯